src/valid/interface.rs

```diff
@@ -440,7 +440,9 @@ impl super::Validator {
                 match types[var.ty].inner {
                     crate::TypeInner::Image { .. }
                     | crate::TypeInner::Sampler { .. }
-                    | crate::TypeInner::BindingArray { .. } => {}
+                    | crate::TypeInner::BindingArray { .. }
+                    | crate::TypeInner::AccelerationStructure
+                    | crate::TypeInner::RayQuery => {}
```

Comment:
Can `RayQuery` handles really be global? I thought they were always local variables.

Reply:
I didn't test this comprehensively. I assume they can be private globals too.